testing/test_end_to_end.py

```diff
@@ -0,0 +1,42 @@
+import unittest
```

Comment:
@KyleBehiels Do this https://github.com/Amplytica/Python_Coding_Standards#file-header-docstring